kafka/consumer-group.go

```diff
@@ -5,54 +5,46 @@ import (
 	"time"
 
 	"github.com/ONSdigital/go-ns/log"
-	"github.com/Shopify/sarama"
 	"github.com/bsm/sarama-cluster"
+	"sync"
 )
 
 var tick = time.Millisecond * 4000
 
+// ConsumerGroup represents a Kafka consumer group instance.
 type ConsumerGroup struct {
 	consumer *cluster.Consumer
 	incoming chan Message
-	closer   chan bool
 	errors   chan error
+	closer   chan bool
```

Comment:
Is there a need for the close channel if there a function that does it?

Reply:
I think so - the closer channel is now used internally to signal the go routine to exit. Open to suggestions of alternatives 👍